nano/node/bootstrap.cpp

```diff
@@ -2436,10 +2436,10 @@ class request_response_visitor : public nano::message_visitor
 		}
 		else if (message_a.response)
 		{
-			auto node_id (message_a.response->first);
-			connection->remote_node_id = node_id;
+			nano::account node_id (message_a.response->first);
```

Comment:
This existed already, but seeing as you are changing the variable to have an explicit type it could be made a const reference to remove a copy.

Reply:
Good idea

Reply:
Can probably change the other one a bit lower as well then